app/webpacker/components/Panel/pages/AnonymizationScriptPage/AnonymizeUser.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Button, List, Message } from 'semantic-ui-react';
+import useLoadedData from '../../../../lib/hooks/useLoadedData';
+import { apiV0Urls, actionUrls, adminAnonymizePersonUrl } from '../../../../lib/requests/routes.js.erb';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+import useSaveAction from '../../../../lib/hooks/useSaveAction';
+
+const ANONYMOUS_ACCOUNT_EMAIL_ID_SUFFIX = '@worldcubeassociation.org';
+const ANONYMOUS_ACCOUNT_NAME = 'Anonymous';
+const ANONYMOUS_ACCOUNT_DOB = '1954-12-04';
+const ANONYMOUS_ACCOUNT_GENDER = 'o';
+const ANONYMOUS_ACCOUNT_COUNTRY_ISO2 = 'US';
```

Comment:
Can you export them from Rails to the frontend using our ugly `wca-data.js.erb` bridge?